DFC.FindACourseClient/Extensions/CourseTypeExtensions.cs

```diff
@@ -0,0 +1,47 @@
+﻿using DFC.FindACourseClient.Models.APIResponses.CourseGet.Enums;
+using DFC.FindACourseClient.Models.ExternalInterfaceModels.Enums;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DFC.FindACourseClient.Extensions
+{
+    public static class CourseTypeExtensions
+    {
+        public static List<DeliveryMode> MapToDeliveryModes(this List<CourseType> courseTypes)
+        {
+            var result = new List<DeliveryMode>();
+
+            foreach (var courseType in courseTypes)
+            {
+                switch (courseType)
+                {
+                    case CourseType.ClassroomBased:
+                        result.Add(DeliveryMode.ClassroomBased);
+                        break;
+
+                    case CourseType.DistanceLearning: // Equivalent doesnt exist in DeliveryMode
```

Comment:
This will be removed, so we do not need to map it

Reply:
Removed this value